independent-publisher-connectors/icanhazdadjoke/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Daniel Laskewitz"
```

Comment:
@Laskewitz Can you please add `stackOwner` property as well..

Documentation - [Adding Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)